Content.Shared/Sticky/Components/IsStuckOnEntityComponent.cs

```diff
@@ -0,0 +1,7 @@
+namespace Content.Shared.Sticky.Components;
+
+[RegisterComponent]
+public sealed class IsStuckOnEntityComponent : Component
```

Comment:
Why do you need this as a separate component? You can use just `StickyComponent.StuckTo != null` to see if sticky item is stuck to something else.

You can add small helper property to `StickyComponent` like:
```
public bool IsStuck => StickyComponent.StuckTo != null;
```

Reply:
wait , is sticky component predicted ?

Reply:
nvm I just needed to move it to shared

Reply:
No, it's server only. Prediction has nothing to do with calculated properties inside component.

Reply:
huh